### PR TITLE
Show which videos have been annotated

### PR DESCRIPTION
**Pull request by manikd31**
This PR corresponds to [this ticket](https://20bn.atlassian.net/browse/AV-4349) and adds a check-mark alongside the videos which have already been annotated, giving the user a better idea of which videos are yet to be annotated.

The layout of the videos-list looks like this:

Screenshot from 2021-02-12 12-13-34

### CODE REVIEW

File: tools/sense_studio/templates/video_list.html [hunk 1]
```diff
@@ -15,14 +15,16 @@ <h1 class="ui header">
...
 	<div class="ui segments">
-	{% for  f, id in folders %}
-		<div class="ui segment">
-			<!-- TODO: Mark videos that were already tagged -->
-			<a href="{{ url_for('annotate', split=split, label=label, path=path, idx=id) }}">
-				<i class="film icon"></i>
-				{{ f }}
-			</a>
-		</div>
+	{% for  f, exists, id in folders %}
```
Comment (corneliusboehm) [inline tools/sense_studio/templates/video_list.html:18]: How about `tagged` instead of `exists`?

**Review by corneliusboehm — COMMENTED** [review]
Working well, but I have a few comments on code style

File: tools/sense_studio/templates/video_list.html [hunk 2]
```diff
@@ -15,14 +15,16 @@ <h1 class="ui header">
...
+	   <div class="ui segment">
+		 <a href="{{ url_for('annotate', split=split, label=label, path=path, idx=id) }}">
```
[the remark this answers is not in the document]
Comment (manikd31) [reply tools/sense_studio/templates/video_list.html:20]: Oh yea, I didn't notice that, thanks. I'll convert it to all-spaces.

File: tools/sense_studio/sense_studio.py [hunk 3]
```diff
@@ -361,12 +361,19 @@ def show_video_list(split, label, path):
     videos = os.listdir(frames_dir)
     videos.sort()
 
+    annotations_exist = []
+    for video in videos:
+        if f'{video}.json' in os.listdir(tags_dir):
+            annotations_exist.append(True)
+        else:
+            annotations_exist.append(False)
```
[the remark this answers is not in the document]
Comment (manikd31) [reply tools/sense_studio/sense_studio.py:369]: Yes, that makes sense. I'll work on this. Thanks!

**Review by corneliusboehm — APPROVED** [review]
:white_check_mark:

**Pull request merged**